torch/csrc/jit/mobile/flatbuffer_loader.h

```diff
@@ -39,13 +45,19 @@ TORCH_API mobile::Module initialize_mobile_module(
 TORCH_API mobile::Module parse_and_initialize_mobile_module(
     std::shared_ptr<char> data,
     size_t size,
+    ExtraFilesMap& extra_files,
     c10::optional<at::Device> device = c10::nullopt);
 
 // Load a mobile::Module from a filepath.
 // This function does steps 1+2+3 described above.
 // We need to have this as a convienience because Python
 // API will need to wrap this. C++ clients should use one
 // versions above.
+TORCH_API mobile::Module load_mobile_module_from_file(
+    const std::string& filename,
+    ExtraFilesMap& extra_files,
```

Comment:
Remove this param. 

User who need extra file shall call parseExtraFiles separately.